packages/eslint-plugin/src/stylex-valid-styles.js

```diff
@@ -2262,6 +2262,7 @@ const pseudoClassesAndAtRules = makeUnionRule(
   makeRegExRule(/^@media/, 'a media query'),
   makeRegExRule(/^@container/, 'a media query'),
   makeRegExRule(/^@supports/, 'a supports query'),
+  makeRegExRule(/^@starting-style/, 'a starting-style query'),
```

Comment:
```suggestion
  makeLiteralRule('@starting-style'),
```

Please change the `RegEx` to a `Literal` here since `@starting-style` is the exact string that must be used is not like `@media` or `@supports` queries where the query is added after the keyword.